doc/genmods.py

```diff
@@ -59,6 +59,7 @@
     ('polymer', 'Polymer models'),
     ('probe', 'Instrument probe'),
     ('profile', 'Model profile'),
+    ('refllib', 'Low level reflectivity calculations'),
```

Comment:
This will pick up the docstrings from the functions defined in refllib but no module level docstrings. This may be good enough for our purposes and we don't need to figure out how to document subpackages "properly". We might drop a comment here to that effect.

Reply:
I had to add all the functions to an __all__ list in refllib.py in order for them to get picked up.  I agree we can figure out how to improve the docstrings in the future.  When you say "module-level docstrings" I'm not sure what you mean - I added a docstring to the top of refllib.py and it seems to have picked that up (it's pretty minimal).  If you have something in mind for a comment/todo to add to the file please feel free to add!